plugins/packs-integrations.js

```diff
@@ -204,10 +206,12 @@ async function pluginPacksAndIntegrationsData(context, options) {
       const packUrl = "v1/packs/";
       const packMDMap = new Map();
       let apiPacksData = [];
+      const packuid = mappedRepos?.[0]?.uid;
       const promises = packDataArr.map((packData) => {
         packMDMap[packData.spec.name] = packData;
         const cloudType = packData.spec.cloudTypes.includes("all") ? "aws" : packData.spec.cloudTypes[0];
-        const url = `${packUrl}${packData.spec.name}/registries/${packData.spec.registries[0].uid}?cloudType=${cloudType}&layer=${packData.spec.layer}`;
+        const packUid = packData.spec.registries.find((registry) => registry.uid === packuid);
+        const url = `${packUrl}${packData.spec.name}/registries/${packUid?.uid || packData.spec.registries[0].uid}?cloudType=${cloudType}&layer=${packData.spec.layer}`;
```

Comment:
@codrin-iftimie say, i have given A, B in the docusaurus config file. So, our conclusion is to use A as the preferred registry to fetch the details of the pack
There can be another scenario, while traversing the list of packs, some packs may be part of registry B, not A. So, this could end up in error if i fetch the details of those packs which are not part of A. Thats the reason why i am creating another packUid here.. i will update the proper comments for this for future correspondence